From 81ecbc5631895a61265a7ef4b1ff226414d8ee23 Mon Sep 17 00:00:00 2001
From: xd1le <elisp.vim@gmail.com>
Date: Tue, 22 Aug 2017 16:16:11 +1000
Subject: [PATCH] remove some BUS stuff

That are not declared for some reason and therefore cause building to
fail.
---
 uinput.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/uinput.cpp b/uinput.cpp
index cbb67bd..c4b1bf3 100644
--- a/uinput.cpp
+++ b/uinput.cpp
@@ -34,8 +34,6 @@ std::map<int, std::string> bus_string = {
     {BUS_GSC, "BUS_GSC"},
     {BUS_ATARI, "BUS_ATARI"},
     {BUS_SPI, "BUS_SPI"},
-    {BUS_RMI, "BUS_RMI"},
-    {BUS_CEC, "BUS_CEC"},
 };
 
 std::map<std::string, int> string_bus = {
@@ -58,8 +56,6 @@ std::map<std::string, int> string_bus = {
     {"BUS_GSC", BUS_GSC},
     {"BUS_ATARI", BUS_ATARI},
     {"BUS_SPI", BUS_SPI},
-    {"BUS_RMI", BUS_RMI},
-    {"BUS_CEC", BUS_CEC},
 };
 
 void print_usage(std::FILE *stream, const char *program) {
-- 
2.14.1

